llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -192,10 +220,11 @@ class Vocabulary {
   /// Helper function to classify an operand into OperandKind
   LLVM_ABI static OperandKind getOperandKind(const Value *Op);
 
-  /// Helpers to return the IDs of a given Opcode, TypeID, or OperandKind
-  LLVM_ABI static unsigned getNumericID(unsigned Opcode);
-  LLVM_ABI static unsigned getNumericID(Type::TypeID TypeID);
-  LLVM_ABI static unsigned getNumericID(const Value *Op);
+  /// Helpers to return the slot index or position of a given Opcode, TypeID, or
```

Comment:
nit: remove the "Helpers..." word. We don't have APIs that are "hinderers" (same goes for "util" names). Just say what it does.